hack/test-cmd.sh

```diff
@@ -19,7 +19,7 @@ function find_tests() {
     local full_test_list=()
     local selected_tests=()
 
-    full_test_list=( $(find "${OS_ROOT}/test/cmd" -name '*.sh') )
+    full_test_list=( $(find "${OS_ROOT}/test/cmd" -name '*.sh' | sort) )
```

Comment:
I prefer unsorted.  Can you randomize instead please.

Reply:
Why? I like knowing how much time I have left in my tests

Reply:
We found a _lot_ of bugs once we stopped sorting in the past, people were not cleaning up cluster resources in test X and not questioning the state of the universe in test X+N, then writing test X+N to only work when run after text X...

Reply:
randomizing without being reproducible leads to flakes that magically pass on a subsequent retry

Reply:
Is that an issue today? You're arguing to change the status quo :)

Reply:
> Is that an issue today?

yeah, hit it while running this suite.

> You're arguing to change the status quo :)

will push a revert with the next batch of changes. I don't think the current random+unreproducible state is helpful, but I don't care enough to argue.